packages/discovery-provider/src/app.py

```diff
@@ -215,7 +215,7 @@ class TimestampJSONEncoder(JSONEncoder):
         def default(self, o):
             if isinstance(o, datetime.datetime):
                 # ISO-8601 timestamp format
-                return o.strftime("%Y-%m-%dT%H:%M:%S Z")
+                return o.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Any danger in changing the expected format here? Could there be other consumers of the data we should check that we're not breaking?